tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/bulkhead/clientserver/Bulkhead5RapidRetry12MethodSynchBean.java

```diff
@@ -0,0 +1,48 @@
+/*
+ *******************************************************************************
+ * Copyright (c) 2017 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package org.eclipse.microprofile.fault.tolerance.tck.bulkhead.clientserver;
+
+import java.time.temporal.ChronoUnit;
+import java.util.concurrent.Future;
+import javax.enterprise.context.ApplicationScoped;
+import org.eclipse.microprofile.fault.tolerance.tck.bulkhead.Utils;
+import org.eclipse.microprofile.faulttolerance.Bulkhead;
+import org.eclipse.microprofile.faulttolerance.Retry;
+import org.eclipse.microprofile.faulttolerance.exceptions.BulkheadException;
+
+/**
+ * A simple method level synchronous @Bulkhead bean that has a retry option, 
+ * with a waitingTaskQueue value that should be ignored.
+ * 
+ * @author Andrew Pielage
+ */
+public class Bulkhead5RapidRetry12MethodSynchBean implements BulkheadTestBackend {
+    
+    @Override
+    @ApplicationScoped
```

Comment:
This is not a producer. Why put ApplicationScoped annotation on?

Reply:
This is not 'new' code. I just renamed it to a more consistent name. The author tag says Andrew Pielage.  If the original ApplicationScvoped came from me IMHO I think it is a good thing..to have some variety and more would be better. We don't have great coverage of scopes (standard or custom) and too many of the test beans have the same scope. For example - do we test that the bulkhead 'in' count is per target bean instance correctly scoped and not per bean class or call site...no, not yet.

Reply:
It is wrong to have that annotation. I

Reply:
Why not? Can't one have fault tolerance to a AppScoped singleton? Please explain? Sure the test does not exploit it but it is a valid thing to have and thus test. Please explain why not.

Reply:
Agh - just spotted it was on the method and not the class and hence a scoping  - apologies. 
 I can only assume that this is a copy and paste error either by Andrew or me. I will remove it with an additional commit tomorrow first thing.